src/cryptoadvance/specter/util/tx.py

```diff
@@ -71,3 +83,45 @@ def decoderawtransaction(hextx, chain="main"):
         ],
     }
     return result
+
+
+def convert_rawtransaction_to_psbt(wallet, rawtransaction) -> str:
```

Comment:
```suggestion
def convert_rawtransaction_to_psbt(wallet_rpc, rawtransaction) -> str:
```
Please change it so that the function directly gets an RPC-object. It might look as it doesn't matter but from an architectual Point of view, it's not so cool to have higher order business-objects (like wallet) in lower level helper-functions.
It would implicitely have a dependency to Wallet which it definitely doesn't have.

Reply:
Done